target-design/TODO

```diff
@@ -0,0 +1,4 @@
+-have not bumped riscv-tools
+-have not fixed boom
+-probably want to bump linux too
+-barstools/mdf are not bumped to master
```

Comment:
Is there a reason to change the barstools pointer unless there are breakages under new firrtl/chisel?